jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -570,7 +570,7 @@ public ReporterConfiguration withSender(SenderConfiguration senderConfiguration)
     private Reporter getReporter(Metrics metrics) {
       Reporter reporter = new RemoteReporter.Builder()
           .withMetrics(metrics)
-          .withSender(senderConfiguration.sender)
+          .withSender(senderConfiguration.getSender())
```

Comment:
oh, come on, Java, it's a private field, why is it visible to a class defined at the same level?